invenio_oaiserver/receivers.py

```diff
@@ -102,3 +103,39 @@ def __call__(self, record, **kwargs):
                 'updated': datetime.now().isoformat(),
             }
         })
+
+
+def update_records():  # records_info):
+    """Update oai record informations."""
+    from invenio_records.models import RecordMetadata
+    #  rinfo = list(records_info)
+    #  if rinfo:
+    records = RecordMetadata.query.all()
```

Comment:
you can use `scan` [1] from `elasticsearch.helpers`.

--
[1] http://elasticsearch-py.readthedocs.org/en/master/helpers.html#scan